src/Tizen.NUI.Components/Controls/Navigation/AppBar.cs

```diff
@@ -0,0 +1,433 @@
+﻿/*
+ * Copyright(c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using Tizen.NUI.BaseComponents;
+using Tizen.NUI.Binding;
+using System.Windows.Input;
+
+namespace Tizen.NUI.Components
+{
+    /// <summary>
+    /// The AppBar class is a class which shows title text and provides navigation
+    /// and action functions on Page.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class AppBar : Control
```

Comment:
Can you check similar class name from other platform?

Reply:
Basically, I refers material design's app bar. (https://material.io/components/app-bars-top)

Class names from other platforms are as follows.
- Flutter : AppBar (https://api.flutter.dev/flutter/material/AppBar-class.html)
- Android : Toolbar (https://developer.android.com/reference/androidx/appcompat/widget/Toolbar)
- IOS : UINavigationBar (https://developer.apple.com/documentation/uikit/uinavigationbar)